<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1845</width>
    <height>1382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="registers">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>20</y>
      <width>251</width>
      <height>1301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>1220</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>1220</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="instructions">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>1081</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>580</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="programCounter">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>570</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textOutput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>631</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>30</y>
      <width>151</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Text Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>120</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Stack</string>
    </property>
   </widget>
   <widget class="QListWidget" name="heap">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>80</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>120</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Heap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heapSize">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>660</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>670</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Heap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="low">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>750</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>850</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>760</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="high">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>840</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="translateButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>1220</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
